core/block.ts

```diff
@@ -190,6 +193,9 @@ export class Block implements IASTNodeLocation, IDeletable {
    * @deprecated August 2019. Use getCommentText instead.
    */
   comment: string|Comment|null = null;
+  /**
+   * @internal
+   */
```

Comment:
I'm not keen on this formatting: there's no reason for it to be three lines instead of one.

But then I had a look at the the original JS version I see that the tooling (or maybe @gonfunko??) has made a bit of a mess of the JSdocs for class fields: sometimes the comment has been attached to the field declaration at the top of the class, and sometimes it remains with the initialisation in the constructor.  Ugh.  That does not need to be dealt with in this PR, but I do think a follow-on to make things as consistent as possible would be desirable.

In the mean time, maybe these unnecessarily-multi-line JSDoc comments are OK where there's going to be a description added?  If so, this PR should probably be consistent—see contrary example below.